tracer/build/_build/Build.Shared.Steps.cs

```diff
@@ -85,27 +85,23 @@ partial class Build
                 var source = NativeProfilerProject.Directory / "bin" / BuildConfiguration / architecture.ToString() /
                              $"{NativeProfilerProject.Name}.dll";
                 var dest = TracerHomeDirectory / $"win-{architecture}";
-                Logger.Info($"Copying '{source}' to '{dest}'");
                 CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
 
                 // Copy native loader assets
                 source = NativeLoaderProject.Directory / "bin" / BuildConfiguration / architecture.ToString() /
                              "loader.conf";
                 dest = MonitoringHomeDirectory;
-                Logger.Info($"Copying '{source}' to '{dest}'");
                 CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
 
                 source = NativeLoaderProject.Directory / "bin" / BuildConfiguration / architecture.ToString() /
                              $"{NativeLoaderProject.Name}.dll";
                 var destFile = MonitoringHomeDirectory / $"{NativeLoaderProject.Name}.{architecture.ToString()}.dll";
-                Logger.Info($"Copying file '{source}' to 'file {destFile}'");
                 CopyFile(source, destFile, FileExistsPolicy.Overwrite);
 
                 source = NativeLoaderProject.Directory / "bin" / BuildConfiguration / architecture.ToString() /
```

Comment:
Since you've already removed the PDB from being added to the MSI, could we continue to copy the PDB to the MonitorinHomeDirectory for local debugging? Or would this keep the PDB included in the `windows-tracer-home.zip`?

Reply:
> Or would this keep the PDB included in the windows-tracer-home.zip?

Yeah, that would include the symbols in there. And I _think_ it's not necessary for local debugging as the symbols _are_ available, you'd just have to specify the location? Not sure how much of an issue that is though - if it's easier to have them in the same folder, we could change how we create `windows-tracer-home.zip` so as to similarly filter out the native symbols. Somewhat preferable if we _don't_ need to do that, but not a big deal

Reply:
After some more thought I think we _should_ do the suggested change (i.e. keep the pdbs in MonitoringHome) for local debugging purposes.

But there's a big refactor of all this coming up shortly when we redo the package layouts etc, and I think we can simplify a lot of the build accordingly. On that basis, I'm going to merge this as-is, and review the process again later (as it's more important that we grab the released native symbols right now IMO)